MultiPoolMiner.ps1

```diff
@@ -568,6 +577,16 @@ while ($true) {
     Start-Sleep $Config.Delay #Wait to prevent BSOD
     $ActiveMiners | Where-Object Best -EQ $true | ForEach-Object {
         if ($_.GetStatus() -ne "Running") {
+		
+            # Launch custom miner variable
+            $MinerProfile = ".\Profile\Start_"+$_.Name+".bat"
```

Comment:
What is this for?

Reply:
launch Start_SgminerKL.bat in profile folder
set ptr 0
i don nt have amd rig to test anymore sad case